Assorted cleanup and updating

iantaylor-NOAA opened the pull request:
This resolves #13, #14, and #17, which were all assigned to me as well as some other minor cleanup such as new name for {pacfintools} and note that we paln to use {asar} in 2025.

@chantelwetzel-noaa, do you have time to quickly look through these edits to make sure I'm on track? I know that we often just push change to the main branch, but I made enough changes that a second set of eyes would be helpful.

iantaylor-NOAA committed:
update survey info

- resolves #13
- adds figure summarizing depth/lat/timing of surveys

iantaylor-NOAA committed:
more edits to data and model choices sections

- add info on projections/forecasts
- change PacFIN.Utilities to pacfintools
- update function for weight-length calcs

iantaylor-NOAA committed:
update some guidance in the running SS3 section

iantaylor-NOAA committed:
minor updates to sections 4 and 5

chantelwetzel-noaa commented:
I will review these changes tomorrow.

iantaylor-NOAA committed:
add update links to SS3 website and User Manual

- forgotten task listed in #17

chantelwetzel-noaa committed:
minor text edits

chantelwetzel-noaa commented on R/survey_summary_US_West_Coast.R:
```diff
@@ -0,0 +1,55 @@
+    nwfscSurvey::pull_catch(
+      common_name = "Pacific spiny dogfish", # random species
+      survey = survey
+    )
```
Should we add the `years` argument here?  The figure includes 1977 Triennial data, but since this year is not recommended for use in assessments (at least not currently), this could cause some confusion.

chantelwetzel-noaa commented on R/survey_summary_US_West_Coast.R:
```diff
@@ -0,0 +1,55 @@
+    ) |> as.factor()
+  )
+
+# transform some dates
```
We should either add an offset to the years for surveys with overlapping years or use different pch options by survey for visibility.

chantelwetzel-noaa committed:
add info to request data

chantelwetzel-noaa reviewed (approved):
All of these changes look great.  I have made some suggestions on the figure but I will leave it up to you to decide whether they need to be done.

chantelwetzel-noaa commented:
While it is not in any of the changes that you made, I was wondering if we should make some modifications to the survey data section.  I think we may want to update survey names in section 1.5 to align with the survey summary pdf document that Stacey created. Thoughts?

iantaylor-NOAA committed:
flag years that we exclude from the survey in plot

iantaylor-NOAA committed:
Merge branch 'updates-from-Ian' of https://github.com/pfmc-assessments/pfmc_assessment_handbook into updates-from-Ian

iantaylor-NOAA commented:
@chantelwetzel-noaa, thanks for looking everything over. Part of my goal of the figure was to illustrate WHY we don't use the early slope survey data because of incomplete spatial coverage, for instance. It's also theoretically possible that someday we could explore the impact of bringing back some of those early years and use sdmTMB to fill in the gaps. Therefore instead of removing those early slope years or the 1977 triennial from the plot, I colored them gray in 364f90b. I tried using different shapes but there are so many points it was a mess, so I've simply shuffled the order of the points so that no one survey completely covers another.

Can you point me to the PDF from Stacey? I'm not tracking it down on Google Drive and the STAR files that were on PAM are no longer available.

chantelwetzel-noaa commented:
The figure looks great.  I just shared the survey document with you.

iantaylor-NOAA committed:
minor revisions to survey names

iantaylor-NOAA committed:
add quotes around doc name

iantaylor-NOAA commented:
Thanks @chantelwetzel-noaa. I changed "AFSC/NWFSC" to "NMFS" for the Triennial to better conform to that doc, and also added a link to the doc, but the names in the doc are already pretty close to what we have and if we shorter our long names to match that doc, they aren't really different from the short versions. And if we lengthen the short versions, they aren't short any more (I would rather use "WCGBTS" than "NWFSC WCGBTS"). Feel free to push further changes to main or a new branch as you see fit, but I'm going to go ahead and merge this branch so we can move on to more important stuff.
